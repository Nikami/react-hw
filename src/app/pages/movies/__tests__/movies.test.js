import React from 'react';
import { shallow } from 'enzyme';

import Movies from '../index';
import SearchBar from '../search-bar';
import SearchList from '../search-list';

describe('DetailsList', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Movies />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has SearchBar', () => {
    expect(wrapper.find(SearchBar).length).toBe(1);
  });

  it('has Footer', () => {
    expect(wrapper.find(SearchList).length).toBe(1);
  });
});
