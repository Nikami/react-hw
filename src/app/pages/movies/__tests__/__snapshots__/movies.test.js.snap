// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsList matches snapshot 1`] = `
<section
  className="movies"
>
  <_default />
  <_default />
</section>
`;
